### ISSUE TITLE
More getaways support

### ISSUE THREAD

**Issue by rubengc** [easydigitaldownloads/EDD-Commissions#166]
Some of my vendors are from Ukranie and Paypal does not allows to send payments to accounts from this country

The most flexible option is set as user fields eddc_account (email or username from the platform) and eddc_getaway(Paypal, Skrill, Stripe, etc)

And allow, on export, filter by the user field eddc_account to separate accounts from differents getaways

**Comment by pippinsplugins** [comment]
We do hope to do this through a more robust payout API we're adding in the future.

**Comment by davidsherlock** [comment]
@pippinsplugins, can MassPay payments be sent directly from the commissions plugin? I noticed the file class-paypal-masspay.php, but wasn't sure how to access this functionality.

**Comment by chriscct7** [comment]
Masspay is not enabled no

**Comment by pippinsplugins** [comment]
@davidsherlock In the future that will be the planned (though it will use the REST API) but not at the moment.

**Comment by rubengc** [comment]
@pippinsplugins can you target this to the next version?

**Comment by pippinsplugins** [comment]
@rubengc I can't say for sure right now.

**Comment by rubengc** [comment]
No news about this?

**Comment by cklosowski** [comment]
I've been working on a Stripe payout in my spare time, but it's not ready for testing yet. I'll keep this topic updated as needed for testing.

**Comment by rubengc** [comment]
But I mean to allow set a desired payout platform without get It integranted through a getaway because much of this services requires a minimum amount to allow you make It automatically

**Comment by joostmq** [comment]
@cklosowski Are you still working on this? With the upcoming EU regulatory changes early next year using a service like Stripe is one of the few ways to avoid getting a very expensive payments institution license (which will then be required for most marketplaces). Would be great to see it added to EDD!

**Comment by davidsherlock** [comment]
@joostmq, what EU changes? can you post a link please.

**Comment by joostmq** [comment]
@davidsherlock https://ec.europa.eu/info/law/payment-services-psd-2-directive-eu-2015-2366_en

A quicker explanation of what's going on: https://medium.com/stories-by-ibanfirst/brace-yourselves-marketplaces-psd2-is-coming-665b2a88c98b

I've also been in touch with the regulatory institution here in The Netherlands and they've confirmed that the platform I'm currently operating (with EDD FES + Commissions) would require a license from 2018 onwards. That's why I've been looking into integration of Stripe Connect, which would take away the obligation to get a license.

**Comment by davidsherlock** [comment]
This could also be mitigated by using PayPal adaptive payments plugins, so the "sale" is directly between the buyer and seller and the marketplace isn't an "agent" to the transaction (simply a platform providing a service - hosting, marketing, etc). You can reverse the roles in the adaptive payments plugin to make the seller a primary recipient in a chained payment for example, so the funds go directly to the seller.

In this instance, the marketplace isn't providing any kind of "payment service" since PayPal is the responsible body, though the complexities of EU VAT MOSS regulations would still need to be considered depending on the structure of the terms of service.

**Comment by joostmq** [comment]
@davidsherlock That's definitely true. But since PayPal is hardly being used by the target audience of my platform (and is not preferred in many countries) supporting Stripe Connect for bank transfers would be a valuable addition to EDD. Previously about 95% of my platform's users preferred bank payments, and I can imagine others are also experiencing the limits of PayPal. The added risk is that as soon as a PayPal payment fails and the amount (temporarily) remains with the platform, you would probably already need a license, whereas with Stripe Connect funds remain in full control of Stripe.

**Comment by cklosowski** [comment]
We're still looking into this. With the addition of the new Stripe Connect APIs they are offering, we can now mimic what we currently do with PayPal.

**Comment by joostmq** [comment]
@cklosowski I'm very happy to hear that, it's a big relief! What kind of integration are you looking at? Would this be using Stripe Connect Custom accounts (considering Standard accounts only work for businesses)?

**Comment by cklosowski** [comment]
@joostmq Yes, as stated above, it would use the Stripe Connect 'managed' accounts, however with the recent changes to Stripe Connect, this is much easier to achieve.

**Comment by pippinsplugins** [comment]
New Stripe Connect request https://secure.helpscout.net/conversation/499018291/73494?folderId=180499
